tests/Feature/Traits/CatchFailures.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Tests\Feature\Traits;
+
+use Illuminate\Testing\TestResponse;
+
+trait CatchFailures
```

Comment:
What exactly is this trait supposed to do? I don't understand what we gain from it. Also see my more specific comments below.

Reply:
When Tests are run and fail with 500 it is quite difficult to figure out what went wrong.
This trait adds a few methods to make it easier to see what happenned.